src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -321,8 +320,9 @@ namespace GraphQL
     public static class StringExtensions
     {
         public static string ToCamelCase(this string s) { }
-        public static string ToFormat(this string format, params object[] args) { }
+        public static string ToConstantCase(this string s) { }
         public static string ToPascalCase(this string s) { }
+        public static string TrimGraphQLTypes(this string name) { }
```

Comment:
It seemed to me that this method (TrimGraphQLTypes) pertains to the library and could be useful, and as it was only a couple lines of code, I left it in the codebase.  However, it is not being used, so we can delete it.

Reply:
50/50